src/coreclr/jit/redundantbranchopts.cpp

```diff
@@ -370,11 +481,26 @@ bool Compiler::optRedundantBranch(BasicBlock* const block)
                         return false;
                     }
 
+                    // Was this an inference from an unrelated relop (GE => GT, say)?
+                    //
+                    const bool domIsUnrelatedRelop = (rii.vnRelation == ValueNumStore::VN_RELATION_KIND::VRK_Unrelated);
```

Comment:
likewise here...if you can rename from `domIsUnrelatedRelop` to `domIsInferredRelop`, that would be great.